src/content/api/loyalty-programs.mdx

```diff
@@ -79,6 +80,12 @@ A loyalty program is a container for one or more [Promotions](/api/promotions),
       The id of the [Media Upload](/api/media-uploads/) image of the Loyalty Program.
     </Property>
   </Properties>
+
+  <Properties heading="Errors">
+    <Error code="403" message="INVALID_ACCOUNT_TYPE">
+      The Account is of the wrong type.
```

Comment:
What is the correct type? You should specify that the account must be type `org` to make this error easier to debug for integrators.

Reply:
A follow-up [PR](https://github.com/centrapay/centrapay-docs/pull/976) is created to update the description